probinet/input/loader.py

```diff
@@ -21,6 +22,40 @@
 from ..models.classes import GraphData
 
 
+def build_adjacency_from_networkx(
+    network: nx.Graph,
+    edge_weight: list[str],
+    file_name: Optional[str] = None,
+    # TODO: add csv file name
+) -> GraphData:
+    """
+
+    Parameters
+    ----------
+
+    Returns
+    -------
+    """
```

Comment:
Could you please fill this section out? Any simple description would work. :smile: